service/controller/chart/v1/key/key.go

```diff
@@ -25,6 +31,20 @@ func ConfigMapNamespace(customResource v1alpha1.Chart) string {
 	return customResource.Spec.Config.ConfigMap.Namespace
 }
 
+func HasForceUpgradeAnnotation(customResource v1alpha1.Chart) (bool, error) {
+	val, ok := customResource.Annotations[ForceHelmUpgradeAnnotationName]
+	if !ok {
+		return false, nil
+	}
+
+	result, err := strconv.ParseBool(val)
```

Comment:
My comment was somehow lost sorry :)

I meant this when this return error? I'd like `HasForceUpgradeAnnotation` to not return error. In general we aim for `key` package functions to not return errors.

Reply:
```
ParseBool returns the boolean value represented by the string. It accepts 1, t, T, TRUE, true, True, 0, f, F, FALSE, false, False. Any other value returns an error
```

Maybe it's ok to return `false` on ParseBool error? I don't know at this point. I leave the decision to you.

Reply:
Yes, I think its best to return false if ParseBool fails. As someone might add the annotation manually and fat finger it.

But I've improved the logs to also show whether force is enabled. As I think that is important for debugging.